clj-refactor.el

```diff
@@ -269,6 +269,16 @@ if it appears to be unused."
 (defvar cljr--find-symbol-buffer "*cljr-find-usages*")
 (defvar cljr--post-command-messages nil "Message(s) to display after the current command is done.")
 
+(defcustom cljr-before-warming-ast-cache-hook (lambda ())
+  "Runs before each time the AST is loaded."
+  :group 'cljr
+  :type 'function)
+
+(defcustom cljr-after-warming-ast-cache-hook (lambda (response))
```

Comment:
Think it's customary to use `nil` as the default here, and then put the `funcall` below in a `when`.

I'm not complete sure, though, but I'd prefer something a little less elegant over a compile warning about unused locals.